website/content/currencies.md

```diff
@@ -7,12 +7,13 @@ anchor: "currencies"
 weight: 230
 ---
 | Code | Num | Exp | Comment                      |
-| ---- | --- | --- | ---------------------------- |
+|------|-----|-----|------------------------------|
```

Comment:
Is this really correct? :thinking: 

https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables

Reply:
No. It's not. I mean it still renders, but my `format-on-save-hook` got the best of me.